tools/README.md

```diff
@@ -88,6 +88,51 @@ part-info --mesh heart.mesh --weights heart.linear.weights \
           --partition heart.linear.metis.part
 ```
 
+### Usage with Docker
+
+Here is a quick walk-through allowing you to launch a container with Coupe bins
+and to interact with it from your host.
+
+```shell
+# Pull docker image.
+docker pull ghcr.io/lihpc-computational-geometry/coupe-x86_64-unknown-linux-musl:main
+
+# Add "coupe" tag to the docker image.
+docker image tag ghcr.io/lihpc-computational-geometry/coupe-x86_64-unknown-linux-musl:main coupe
+
+# Run the container while binding the current directory and the container.
+# This will allow you to access inner generated data from your host.
+docker container run -dit \
+    --name coupe_c \
+    --mount type=bind,source="$(pwd)",target=/coupe/shared \
+    coupe
+
+# Generate a linear weight distribution from an embedded mesh file.
+docker exec coupe_c sh -c 'cat meshes/hole.mesh | weight-gen \
```

Comment:
Can we run the `cat` outside the container ?
The command is else kind of misleading as mesh must be inside the bind mount!